src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -4946,8 +4946,7 @@ namespace ZoneEquipmentManager {
         auto &thisZoneEquip(ZoneEquipConfig(ZoneNum));
         int numRetNodes = thisZoneEquip.NumReturnNodes;
         Real64 totReturnFlow = 0.0; // Total flow to all return nodes in the zone (kg/s)
-        Real64 totVarReturnFlow =
-            0.0; // Total variable return flow, for return nodes connected to an airloop with an OA system or not with specified flow (kg/s)
+        Real64 totVarReturnFlow = 0.0; // Total variable return flow, for return nodes connected to an airloop with an OA system or not with specified flow (kg/s)
```

Comment:
Unrelated whitespace-only change, just got pulled in during a merge of some other work by accident.